config/cloudinary.yml

```diff
@@ -0,0 +1,19 @@
+---
+development:
+  cloud_name: sasaranandrei
+  api_key: '788924322271915'
+  api_secret: ZIbGl7lULuQlx8Y4WgQhjzfQ6ao
```

Comment:
the secrets should not be committed to the repo, instead you should use environment variables (via dotenv gem) or use Rails credentials